lib/protocol/events.js

```diff
@@ -203,12 +203,13 @@ function eventsProtocolFactory (
     };
 
 
-    EventsProtocol.prototype.publishGraphStarted = function (graphid, status, nodeId) {
+    EventsProtocol.prototype.publishGraphStarted = function (graphid, data, nodeId) {
         var self = this;
         assert.uuid(graphid);
-        assert.string(status);
-
-        var data = { status: status };
+        assert.object(data);
+        assert.uuid(data.graphId);
```

Comment:
For this function, we should check `graphId, graphName and status` to keep the correct usage.